manifests/core-operator/all-core-operator-resources.yaml

```diff
@@ -671,10 +672,10 @@ spec:
           protocol: TCP
         resources:
           limits:
-            cpu: 1000m
+            cpu: 500m
             memory: 300Mi
           requests:
-            cpu: 500m
+            cpu: 300m
```

Comment:
How about set the CPU request the same with the CPU limit, and that gets assigned a QoS class of Guaranteed.